src/modules/decklink/consumer_decklink.cpp

```diff
@@ -380,7 +381,23 @@ class DeckLinkConsumer
 		// Initialize members
 		m_count = 0;
 		preroll = preroll < PREROLL_MINIMUM ? PREROLL_MINIMUM : preroll;
-		m_channels = mlt_properties_get_int( properties, "channels" );
+		m_inChannels = mlt_properties_get_int( properties, "channels" );
+		if( m_inChannels <= 2 )
+		{
+			m_outChannels = 2;
+		}
+		else if( m_inChannels <=4 )
```

Comment:
The docs say "only 2, 8 or 16 channel output is supported." and indeed it fails on 4 for me in testing.

BTW my SDI to HDMI mini converter does multi-channel out as my receiver displays something about the number of channels, but it has a strange output with mixing channels because it is configured for 5.1 and thinks the input is 6.1 perhaps due to some incompatible signaling.